bin/local_build_native.sh

```
@@ -0,0 +1,106 @@
+## -- SETUP -- ##
+
+__dirname="$(CDPATH= cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# by default, workdir is the parent directory of the cloned repo
+workdir="${workdir:-"${__dirname}/../.."}"
+
+recipe=""
+fullversion=""
+remote_host="192.168.1.185"
+remote_user="poddingue"
+remote_dir="~/nodejs-builds"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)